src/launcher/java/org/truffleruby/launcher/RubyLauncher.java

```diff
@@ -59,6 +59,9 @@ protected void printVersion() {
 
     @Override
     protected List<String> preprocessArguments(List<String> args, Map<String, String> polyglotOptions) {
+        // This is the last place we can set a system property before the Ruby language is created
+        System.setProperty("truffleruby.single_threaded", Boolean.FALSE.toString());
```

Comment:
Shouldn't this also be enabled in `Launcher`?
I think it could be just in a static initializer in `Launcher`.

Reply:
Isn't `Launcher` ever part of other SVM images that may actually use another language's launcher?

Reply:
Indeed, it probably is for polyglot images.
Then in `runRubyMain` before `createContext()` sounds best.

Reply:
Or in createContext() itself would also be a good place, along other options disabled if we have a real ruby launcher.